test-utils/runtime/src/lib.rs

```diff
@@ -217,6 +217,8 @@ decl_runtime_apis! {
 		fn do_trace_log();
 		/// Verify the given signature, public & message bundle.
 		fn verify_ed25519(sig: ed25519::Signature, public: ed25519::Public, message: Vec<u8>) -> bool;
+		/// Write the given `value` under the given `key` into the storage and then optional panic.
+		fn write_key_value(key: Vec<u8>, value: Vec<u8>, panic: bool);
```

Comment:
I think it'd be a little clearer to have two methods here instead of the mystery meat `bool` argument, e.g.

```suggestion
		// Write the given `value` under the given `key` into the storage.
		fn write_key_value(key: Vec<u8>, value: Vec<u8>);
		// Write the given `value` under the given `key` into the storage and then panic.
		fn write_key_value_and_panic(key: Vec<u8>, value: Vec<u8>);
```

Reply:
I also thought about this, but this is just for tests. So, I don't see any real reason for this.